File: src/main/java/top/dsbbs2/bukkitcord/api/IPluginDescription.java
package top.dsbbs2.bukkitcord.api;

public interface IPluginDescription {
    String getName();
    String getMain();
    String getVersion();
    List<String> getAuthor();
    Set<String> getDepends();
    Set<String> getSoftDepends();
    String getDescription();
    Object getDelegate();
}

File: src/main/java/top/dsbbs2/bukkitcord/api/IPluginDescription.kt
package top.dsbbs2.bukkitcord.api

interface IPluginDescription {
    val name: String?
    val main: String?
    val version: String?
    val author: List<String?>?
    val depends: Set<String?>?
    val softDepends: Set<String?>?
    val description: String?
    val delegate: Any?
}